shared_model/interfaces/signable.hpp

```diff
@@ -61,4 +61,18 @@ namespace shared_model {
     };
   }  // namespace interface
 }  // namespace shared_model
+
+namespace std {
+  template <>
+  struct hash<shared_model::detail::
+                  PolymorphicWrapper<shared_model::interface::Signature>> {
+    size_t operator()(
+        const shared_model::detail::
+            PolymorphicWrapper<shared_model::interface::Signature> &sig) const {
+      return hash<std::string>()(sig->publicKey().blob()
+                                 + sig->signedHash().blob());
```

Comment:
I guess, better to use boost hash combine for it. [Check out](http://www.boost.org/doc/libs/1_35_0/doc/html/hash/combine.html) it.